extras/usd/tutorials/endToEnd/tutorial_scripts/add_shadingVariants.py

```diff
@@ -93,6 +93,7 @@ def _AddShadingToBall(stage):
         with shadingVariant.GetVariantEditContext():
             whichBall = variantName.split('_')[-1]
             texPath = os.path.join(texDir, 'ball%s.tex' % whichBall)
+            texPath = texPath.replace('\\', '/')
             # in the current variant, modify the color
```

Comment:
Might be an idea here to expose the path normalizer you wrote earlier via Python and call that here instead ~ then normalization will behave the same everywhere. Then it would cover drive letter upper-lower-casing etc...